### PR TITLE
[374] Grammar edit readme.md

### LINKED ISSUE #374: Grammar edit the readme
### Description

Creating this issue to capture minor copy-edits I did while reviewing this library in preparation for adding it to the docs site :)

### PR DESCRIPTION
**Pull request by celeste-horgan**
#### Summary

Grammar/wording edits to some slightly awkward language in your readme. Doing this in prep for linking this on the docs site! Quality for all!

#### Relevant Issues

Closes #374. 

#### Todo

- Tests
    - [ ] Unit 
    - [ ] Integration
- [ ] Documentation
- [ ] Add Release Notes entry.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,49 +39,58 @@ Currently this library supports synchronising the following entities in commerce
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 ## Usage
 
-commercetools sync is a Java library that could be used to synchronise CTP data in any of the following ways:
+commercetools sync is a Java library that synchronises commercetools platform data in the following ways:
```
Comment (butenkor) [inline README.md]: synchronises or better `imports`?

File: README.md [hunk 2]
```diff
@@ -9,10 +9,9 @@
 
 More at https://commercetools.github.io/commercetools-sync-java
  
-Java Library used to import and/or sync (taking care of changes) data into one or more commercetools projects from external sources such 
-as CSV, XML, JSON, etc.. or even from an already existing commercetools project.
+Java jibrary for importing and syncing (taking care of changes) data into one or more commercetools projects from external data files (CSV, XML, JSON) or from another commercetools project.
```
Comment (butenkor) [inline README.md]: jibrary?

File: README.md [hunk 3]
```diff
@@ -40,49 +39,58 @@ Currently this library supports synchronising the following entities in commerce
...
 
-> Synchronise: Resources will either be created or updated. But they will **not** be deleted.
+> **Note**: During a synchronisation, resources are either be created or updated, but **not** deleted.
 
-⚡ Take a look at the [Quick Start Guide](/docs/usage/QUICK_START.md) to find out how to build a product importer in a glance!
+⚡ See the [Quick Start Guide](/docs/usage/QUICK_START.md) for more information on building a product importer!
 
 ### Prerequisites
  
  - Make sure you have `JDK 8` installed.
```
[the remark this answers is not in the document]
Comment (heshamMassoud) [reply README.md]: and regarding the specific jdk8u[tag], I highly doubt that (but can't confirm) there is a breaking change in the jdk8 updates that could break the implementation of the library. 

I think JDK 8 is enough, I haven't seen it before in java libraries to document the jdk version with the tag and build.

[the remark this answers is not in the document]
Comment (celeste-horgan) [reply README.md]: Okay then. Let's leave it as is :)

File: README.md [hunk 4]
```diff
@@ -40,49 +39,58 @@ Currently this library supports synchronising the following entities in commerce
...
 
-commercetools sync is a Java library that could be used to synchronise CTP data in any of the following ways:
+commercetools sync is a Java library that synchronises commercetools platform data in the following ways:
 
 1. Synchronise data coming from an external system in any form (CSV, XML, etc..) that has been already mapped to 
 [commercetools-jvm-sdk](https://github.com/commercetools/commercetools-jvm-sdk) resource draft objects 
 (e.g. [CategoryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/categories/CategoryDraft.java)).
 
-2. Synchronise data coming from an already-existing commercetools project in the form of 
+2. Synchronise data coming from another commercetools project as
```
[the remark this answers is not in the document]
Comment (celeste-horgan) [reply README.md]: Best solved by just removing the word, I think ;)

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (celeste-horgan) [reply README.md]: Then let's be nonspecific :)

**Comment by celeste-horgan** [comment]
Sorry it took me so long to revisit this! I got busy onboarding a new colleague :)

**Pull request merged**